simple-todo/package.json

```diff
@@ -4,22 +4,23 @@
   "author": "Paul Adams",
   "private": true,
   "scripts": {
-    "dev": "webpack-dev-server --inline --hot",
-    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules"
+    "dev": "webpack-dev-server --mode=development --inline --hot",
+    "build": "webpack --mode=production --progress --hide-modules"
   },
   "dependencies": {
-    "vue": "^2.0.0-beta.5",
-    "vuex": "^2.0.0-rc.5"
+    "vue": "^2.5.17",
+    "vuex": "^3.0.1"
   },
   "devDependencies": {
-    "babel-core": "^6.0.0",
-    "babel-loader": "^6.0.0",
-    "babel-preset-es2015": "^6.13.2",
-    "cross-env": "^1.0.6",
-    "css-loader": "^0.23.1",
-    "file-loader": "^0.8.4",
-    "vue-loader": "^9.2.2",
-    "webpack": "^2.1.0-beta.20",
+    "@babel/core": "^7.1.2",
```

Comment:
@babel represents the latest namespace. I believe @vue is in the process of migrating to this approach as well, but -- as of this PR -- has only transitioned some of the CLI packages.